### DISCUSSION TITLE
Rethinking Pricing Model of infracost Cloud

### DISCUSSION THREAD

**Discussion by thundering-herd** [infracost/infracost#2375] [General]
Hej!

I hope you're doing well and having a great day! I wanted to reach out and share some feedback on your pricing model - specifically the payment structure for pull request authors.

While I think your service is great, I've noticed that the current pricing model is a bit tough to swallow. If only one person creates one pull request per month, my bill skyrockets faster than Elon Musk's net worth!

I was wondering if you guys have any internal discussions going on about this, and if you'd be open to rethinking the pricing model to reflect the number of pull requests created instead of the number of authors. This approach would make things much more manageable on my end and keep my budget from feeling like a one-way ticket to Mars :)

Anyway, thanks for taking the time to hear me out, and I look forward to hearing back from you soon.

Cheers

**Comment by alikhajeh1** [comment]
@thundering-herd feedback is always welcome! My co-founder, @hassankhosseini, will follow-up with you soon as he's been talking to lots of users about pricing :)

**Comment by hassankhosseini** [comment]
Hey Camal (@thundering-herd),
Thanks for opening this issue. First off, I love your examples 🌎 🚀 (I couldn't find an Elon emoji or a mars one) haha, and sorry about the delayed response.

Pricing is hard, sorry I haven't made one that doesn't match up with your setup (yet!). One of our principles is to be open, so let me write out my thoughts. Sorry about the long response coming 🥲

I have four principles for pricing:
1. Is it measurable
2. Is it easy and understandable (can the user do the math in their head)
3. Is it easy for procurement to understand and forecast
4. It should not limit usage

We tested a few different pricing approaches, but all failed one or two principles, and in turn we failed to get users to success. The one that has worked the best so far is the per seat pricing model. It's easy to understand, do the maths in your head and measure, and the companies roughly know how many seats they need (we can even give you a script to run), and when it comes to procurement teams, they can forecast out for the next 1-3 years how many seats they need as it aligns with hiring strategy. Most importantly, it doesn't limit the usage (get as many cost estimates as you want: don't just deploy it for prod, get it in every environment and every repo you have, do what-if comparisons, send for approvals with different setups to JIRA etc).

But we are a startup, and so we never stop testing. Would you be up for a zoom call about per-price-estimate pricing? I want to dig into some of the details, and maybe you'll be our first test customer on a new pricing model :) my email is hassan.hosseini@infracost.io

**Comment by hassankhosseini** [comment]
I'll close this out for now, but please reach out to me and we can discuss pricing and what I can do to help in your use-case 👍
